### ISSUE TITLE
Root age never modified

### ISSUE THREAD

**Issue by Anaphory** [NBISweden/MrBayes#187]
### What is the current observed behaviour?

The root age of the result trees is always 2400 years

### What is the expected/wanted behaviour?

The root age of the result trees should be a priori uniformly distributed between 1600 years and 4000 years, and definitely not be fixed.

(In general, for a uniform distribution `prset treeagepr = uniform(min, max)`, the root age seems to be `minimum(1.5*min, max)`)

### How may we reproduce this bug?

Run MrBayes on the attached Nexus file

### Would you be able to compile and run MrBayes to test fixes to this bug?
  - [X] Yes, I know how to use `git` and how to compile MrBayes.

### What is the environment that you run MrBayes in?
  - Operating system (including variant and release): Linux 5.7.0-1-MANJARO #1 SMP PREEMPT Mon May 11 00:19:50 UTC 2020 x86_64 GNU/Linux
  - Version of MrBayes: MrBayes 3.2.7 x86_64
  - If possible, include the output of the `Version` command in MrBayes below:
```
   ---------------------------------------------------------------------------
   Version

   MrBayes 3.2.7

   Features:  SSE AVX FMA readline
   Host type: x86_64-unknown-linux-gnu (CPU: x86_64)
   Compiler:  gnu 9.3.0
   ---------------------------------------------------------------------------

```
### Other information that may be of use to us in resolving this issue
It looks like there may be operators missing changing the root age.
Tests with other non-fixed distributions, and maybe a reduced example, will follow.
[CaribanMB.txt](https://github.com/NBISweden/MrBayes/files/4757742/CaribanMB.txt)
[OutputWithRootAge2400y.pdf](https://github.com/NBISweden/MrBayes/files/4757759/OutputWithRootAge2400y.pdf)
[bugreport.zip](https://github.com/NBISweden/MrBayes/files/4757766/bugreport.zip)

**Comment by Anaphory** [comment]
Node operators (operators with V as argument) are actually added even in this case:

      The MCMC sampler will use the following moves:
         With prob.  Chain will use move
            0.79 %   Dirichlet(Pi)
            0.79 %   Slider(Pi)
           15.87 %   ExtSprClock(Tau,V)
           19.05 %   NNIClock(Tau,V)
           12.70 %   ParsSPRClock(Tau,V)
           31.75 %   NodesliderClock(V)
            4.76 %   TreeStretch(V)
            4.76 %   Multiplier(Net_speciation)
            4.76 %   Slider(Relative_extinction)
            4.76 %   Slider(Relative_fossilization)

This error persists with other tree age priors, but setting the clock rate to a non-fixed prior makes the  model work.

**Comment by zhangchicool** [comment]
If the clock rate prior is not set, the default setting is fixed(1.0). This is absolutely a wrong model given the tip/node ages are hunderds or thousands.  
MrBayes gives a warning for this:
      WARNING: You have calibrated the tree but the clock rate is fixed to 1.0.
         This means that time is measured in expected changes per time unit. If
         the calibrations use a different time scale, you need to modify the model
         by introducing a prior for the clock rate ('prset clockratepr').
Eventually, the moves changing the node ages are rejected so that the root age is unchanged. As you have noted, setting a proper prior for the clock rate dismissed this problem. 
This is not a bug but rather a model misspecification.

> This error persists with other tree age priors, but setting the clock rate to a non-fixed prior makes the model work.

Another warning which is irrelevant and does not affect anything but worth noting is that  
      WARNING: There is one character incompatible with the specified
               coding bias. This character will be excluded.
